restart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ENGLISH">
+<head>
+    <meta charset="UTF-8">
+    <title> Becks Webpage</title>
+    <link rel="stylesheet" href="restart.css" type="text/css"/>
+</head>
+<style> 
+h1,h2,h3 {text-align: center;}
+p {text-align: left;}
+</style>
+<body>
+
+<div id="main">
+<h1 id="title"> Becks:
+</h1>
+
+ <p id="def"> 
+    /bɛks/ <i>noun</i> NORTHERN ENGLISH </p>
+     <p id="def"><em>
```

Comment:
Subjective note. Em tag is used to define emphasised text by italicising. This paragraph has been styled in the CSS so it could be italicised there and em element removed.